src/app/components/ATIAnalytics/params/index.ts

```diff
@@ -90,7 +86,7 @@ const pageTypeUrlBuilders = {
   [MOST_WATCHED_PAGE]: buildMostWatchedATIUrl,
   [INDEX_PAGE]: buildIndexPageATIUrl,
   [FEATURE_INDEX_PAGE]: buildIndexPageATIUrl,
-  [TOPIC_PAGE]: buildTopicPageATIUrl,
+  [TOPIC_PAGE]: noOp,
```

Comment:
What's the point the of the noOp here? I guess the main question is what are the differences between these [2 params](https://github.com/bbc/simorgh/blob/1d1a48d018b57ecbb41b6dcaa4b6a5b764e157e8/src/app/components/ATIAnalytics/params/index.ts#L215-L216), what are the situations where data is passed in instead of atiData for the migrated pages?

Reply:
The `noOp` just returns an empty object for the current `TOPIC_PAGE` param builder being used until we clean up that params folder to get rid of the legacy strategy pattern. Eventually `genericPage` will be used for all page types. Does that make sense?